dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
As far as I understand you need RewriteBase there, because the default is broken.

And, assuming that the /library/ and /library-prerelease/ rules are meaningfully different, we can't hard-code one here, because  this file will be copied to both directories.